### PR TITLE
Add possibility to remove controller

### PR DESCRIPTION
**Pull request by EugeneButusov**
In order of needs in some of projects, and to support undo-ability, I offer to add controllers removing methods.

### CODE REVIEW

File: lib/controller.js [hunk 1]
```diff
@@ -183,6 +183,38 @@ class Controller extends DataService {
...
+          });
+        }
+      } catch (err) {
+        if (this.log) {
+          this.log.error(
+            `Unset route for action: ${action.name} and path ${this.path}/${action.path}`);
+          this.log.error('Error', err);
+        } else {
+// eslint-disable-next-line no-console
+          console.log(`Unset route for action: ${action.name} and path ${this.path}/${action.path}`);
```
Comment (vedi) [inline lib/controller.js]: maybe, `console.error`?

File: lib/controller.js [hunk 2]
```diff
@@ -183,6 +183,38 @@ class Controller extends DataService {
...
+          }
+          action.method.forEach((method) => {
+            action.transports.forEach(
+              transport => transport.removeRoute(method, this.path, action));
+          });
+        }
+      } catch (err) {
+        if (this.log) {
+          this.log.error(
+            `Unset route for action: ${action.name} and path ${this.path}/${action.path}`);
```
Comment (vedi) [inline lib/controller.js]: Change to "Cannot unset" as a message about an error, but not a process.

File: lib/controller.js (hunk 1 above)
Comment (EugeneButusov) [inline lib/controller.js]: fixed, and the same in `bind()` method.

File: lib/controller.js (hunk 2 above)
Comment (EugeneButusov) [inline lib/controller.js]: fixed, and the same in `bind()` method.

File: lib/controller.js [hunk 3]
```diff
@@ -169,13 +169,45 @@ class Controller extends DataService {
...
+        throw err;
+      }
+    });
+  }
+
+  unbind() {
+    if (typeof (this.path) === 'string') {
+      this.path = [this.path];
+    }
+    _.forEach(_.sortBy(this.actions, 'priority'), (action) => {
```
Comment (mstmustisnt) [inline lib/controller.js]: just wonder why you use lodash forEach here. and native forEach later

File: lib/controller.js [hunk 4]
```diff
@@ -169,13 +169,45 @@ class Controller extends DataService {
...
+      try {
+        if (action.enabled) {
+          if (typeof (action.method) === 'string') {
```
Comment (mstmustisnt) [inline lib/controller.js]: you don't need parentheses with `typeof`

File: lib/controller.js (hunk 3 above)
Comment (EugeneButusov) [inline lib/controller.js]: it's copy-paste of `bind()` method. Will refactor both of them.

File: lib/controller.js (hunk 4 above)
Comment (EugeneButusov) [inline lib/controller.js]: it's copy-paste of `bind()` method. Will refactor both of them.

Comment (EugeneButusov) [inline lib/controller.js]: done.

File: lib/controller.js (hunk 3 above)
Comment (EugeneButusov) [inline lib/controller.js]: done.

**Pull request merged**